end_app/weather_app/entities.py

```diff
@@ -103,25 +103,143 @@ def from_raw_forecast_data(cls, forecast_data):
+                qualitative_name = 'Very Poor'
+            return '0-15400μg/m3', qualitative_name
+        elif value == 'no':
+            if self.no < 100:
+                qualitative_name = 'Good'
+            else:
+                qualitative_name = 'Very Poor'
+            return '0-100ppb', qualitative_name
+        elif value == 'no2':
+            if self.no2 < 40:
+                qualitative_name = 'Good'
+            elif self.no2 < 70:
+                qualitative_name = 'Fair'
+            elif self.no2 < 150:
+                qualitative_name = 'Moderate'
+            elif self.no2 < 200:
+                qualitative_name = 'Poor'
+            else:
+                qualitative_name = 'Very Poor'
+            return '0-200ppb', qualitative_name
+        elif value == 'o3':
+            if self.o3 < 60:
+                qualitative_name = 'Good'
+            elif self.o3 < 100:
+                qualitative_name = 'Fair'
+            elif self.o3 < 140:
+                qualitative_name = 'Moderate'
+            elif self.o3 < 180:
+                qualitative_name = 'Poor'
+            else:
+                qualitative_name = 'Very Poor'
+            return '0-180ppb', qualitative_name
+        elif value == 'so2':
+            if self.so2 < 20:
+                qualitative_name = 'Good'
+            elif self.so2 < 80:
+                qualitative_name = 'Fair'
+            elif self.so2 < 250:
+                qualitative_name = 'Moderate'
+            elif self.so2 < 350:
+                qualitative_name = 'Poor'
+            else:
+                qualitative_name = 'Very Poor'
+            return '0-350ppb', qualitative_name
+        elif value == 'pm2_5':
+            if self.pm2_5 < 10:
+                qualitative_name = 'Good'
+            elif self.pm2_5 < 25:
+                qualitative_name = 'Fair'
+            elif self.pm2_5 < 50:
+                qualitative_name = 'Moderate'
+            elif self.pm2_5 < 75:
+                qualitative_name = 'Poor'
+            else:
+                qualitative_name = 'Very Poor'
+            return '0-75μg/m³', qualitative_name
+        elif value == 'pm10':
+            if self.pm10 < 20:
+                qualitative_name = "Good"
+            elif self.pm10 < 50:
+                qualitative_name = "Fair"
+            elif self.pm10 < 100:
+                qualitative_name = "Moderate"
+            elif self.pm10 < 200:
+                qualitative_name = "Poor"
+            else:
+                qualitative_name = "Very Poor"
+            return "0-200μg/m³", qualitative_name
+        elif value == "nh3":
+            if self.nh3 < 200:
+                quantitative_value = "Good"
+            else:
+                quantitative_value = "Very Poor"
+            return "0-200ppb", quantitative_value
+
+    def to_list(self):
+        result = []
+        for field in fields(self):
+            field_name = field.name
+            value = getattr(self, field_name)
```

Comment:
nie da się użyć lepszej wartości niż value? Jeśli coś opisuje jaka to wartość to już będzie to lepsze